### DISCUSSION TITLE
傀影肉鸽总是选择大锁结局

### DISCUSSION THREAD

**Discussion by mj2337** [MaaAssistantArknights/MaaAssistantArknights#3191] [Q&A]
在不期而遇节点如果遇到大锁的分支，会选择拿小熊，但是在后续的关卡中又不拿剪刀，而且MAA好像还没写好大锁该怎么打，所以有时候天选打到结局却过不去。想改一下以后默认不拿小熊，但是没有找到在哪里改，不知道有没有大佬指点一下

**Comment by MistEO** [comment]
@WWPXX233 我们要不要适配一下大锁啊（

**Reply by WWPXX233** [reply]
没有几个关于大锁的藏品正常打很费力尤其是哪个让大锁禁止回血的 （全图aoe 伤害很高 我觉得还是欺负影子去吧

**Reply by mj2337** [reply]
所以大佬怎么改才能不拿小熊呀，我没找到相关文件

**Reply by ntgmc** [reply]
这个好像没法改 maa默认要拿 又没有json决定不期而遇的选择
